### PR TITLE
Make repo structure

### PR DESCRIPTION
**Pull request by Ivlo**
Issue #1

I´ve made the structure of our frontend-framework for this i´ve added:

- Npm  
- webpack  
- less  
- postcss(autoprefixer)  

You can view all our dependencies in our package.json.

I´ve made a example.less to test that everything it´s working ok,now we can build our css with `npm run build`. This generate our file dist/style.css with all our styles prefixed.....

You can view our webpack config in `webpack.config.js`

@tomasperezv @jobandtalent/frontend plz give it a look when you can :)

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+var ExtractTextPlugin = require("extract-text-webpack-plugin");
+var autoprefixer = require('autoprefixer');
+
+module.exports = {
+  entry: [
+    './source/stylesheets/colors.less',
+    './source/stylesheets/typography.less',
+    './source/stylesheets/example.less'
```
Comment (tomasperezv) [inline webpack.config.js]: I'm not following why we'd need to add the `example.less` here.

Comment (Ivlo) [inline webpack.config.js]: it´s not necessary. I´ve made this file to test that everything it was working ok(autoprefixer and less vars)

Comment (tomasperezv) [inline webpack.config.js]: @Ivlo then just remove it ;)

File: source/stylesheets/colors.less [hunk 2]
```diff
@@ -0,0 +1,63 @@
+//----COLORS----------
```
Comment (montiu) [inline source/stylesheets/colors.less]: I will remove this kind of comments, captain obvious

File: source/stylesheets/colors.less [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+@ultra-low-light-text:rgba(255,255,255,0.6);
+
+//Positive
+@high-positive:#227321;
+@positive:rgba(42,128,40,1);
+@low-positive:rgba(42,128,40,0.4);
+@ultra-low-positive:rgba(42,128,40,0.08);
+
+//Warning
+@high-warning:#FF9C00;
```
Comment (montiu) [inline source/stylesheets/colors.less]: we will use rgba or hex ?

File: source/stylesheets/typography.less [hunk 4]
```diff
@@ -0,0 +1,119 @@
+// Weight
+
+.bold {
+  font-weight:700;
```
Comment (montiu) [inline source/stylesheets/typography.less]: no space after : ?

File: source/stylesheets/colors.less (hunk 3 above)
Comment (calderon) [inline source/stylesheets/colors.less]: Not sure if our current less version has this feature, but with SASS we can do:

$high-light-text: rgb(255,255,255);
$light-text: rgba($high-light-text, .8);

I would bet for that if possible.

Any case, +1 to change that warning color for rgb.

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+var ExtractTextPlugin = require("extract-text-webpack-plugin");
```
Comment (calderon) [inline webpack.config.js]: Use ' insead "

File: src/stylesheets/colors.less [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+@ultra-low-warning: rgba(255, 168, 0, 0.08);
+
+// Negative/Destructive
+@high-negative: #E04747;
+@negative: rgba(229, 83, 83, 1);
+@middle-negative: rgba(229, 83, 83, 0.6);
+@low-negative: rgba(229, 83, 83, 0.4);
+@ultra-low-negative: rgba(229, 83, 83, 0.08);
+
+@benefit-title: rgb(93, 109, 121);
```
Comment (alejandroweb) [inline src/stylesheets/colors.less]: I would remove benefit colors :dancers:

Comment (tomasperezv) [inline src/stylesheets/colors.less]: @alejandroweb code specifics on the less fw are not in the scope of this change, let's do it in a separate PR.
